app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>MyProject1</title>
+
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
` Ruby on Rails Tutorial Sample App` 
tìm hiểu và sử dụng i18n cho toàn bộ text ở tất cả các file html nhé